doc/distrib/NodeHelpFiles/_review/Revit.Elements.ElementType.Duplicate.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`ElementType.Duplicate` attempts to duplicate the given element with the given name. If the name is already taken an error will be presented on the node.
```

Comment:
"type" is missing here-attempts to duplicate the given element "type" with the given name